nni/algorithms/compression/pytorch/pruning/auto_compress_pruner.py

```diff
@@ -34,6 +34,8 @@ class AutoCompressPruner(Pruner):
         Function used for the first subproblem of ADMM Pruner.
         Users should write this function as a normal function to train the Pytorch model
         and include `model, optimizer, criterion, epoch` as function arguments.
+    criterion: function
+        Function used to calculate the loss between the target and the output. By default, we use CrossEntropyLoss.
```

Comment:
so it should be a function not an instance? so users should provide `loss` for criterion, where `loss = nn.CrossEntropyLoss()`? then it is better to give this concrete example

Reply:
add doc str